packages/netlify/src/index.ts

```diff
@@ -237,18 +249,37 @@ export default function netlifyIntegration(
 		}
 	}
 
+	async function getFilesByGlob(
+		include: Array<string> = [],
+		exclude: Array<string> = []
+	): Promise<Array<URL>> {
+		const files = await glob(include, {
+			cwd: fileURLToPath(rootDir),
+			absolute: true,
+			ignore: exclude,
+		});
+		return files.map((file) => pathToFileURL(file));
```

Comment:
It may be an issue. But I think if `include` is empty, then no glob is executed, which means that the `exclude` won't be picked up. I tried to look in the `fast-glob` docs, but I couldn't find anything.

Reply:
It's handled [here](https://github.com/withastro/adapters/blob/include-files/packages/netlify/src/index.ts#L275-L286), where the excluded files are globbed and passed to `copyDependenciesToFunction`, which skips any that match. This means it excludes files that are automatically traced as well as ones included manually.